### ISSUE TITLE
Bug running altered-temperature cross-sections with MPI

### ISSUE THREAD

**Issue by geogunow** [mit-crpg/openmc#868]
I have encountered a small bug running constant-temperature cross-sections at a non-default temperature in OpenMC. Specifically, if I take a simple example from OpenMC, say the pin-cell, and add the following lines to the `settings.xml` file:

```
    <ptables>true</ptables>
    <temperature_default>566.483</temperature_default>
    <temperature_method>interpolation</temperature_method>
    <temperature_tolerance>300</temperature_tolerance>
```
then OpenMC runs just fine running normally with the simple openmc command. However, when I do:
```
mpirun -np 2 openmc
```
I get an error and OpenMC fails to run. Specifically, it complains about the temperature for cross-sections:
```
 WARNING: Cross sections for U234 are only available at one temperature.
          Reverting to nearest temperature method.
 ERROR: Nuclear data library does not contain cross sections for U234 at
        temperatures that bound 566 K.
```
Again, this error does not show up when running without MPI. It also does not show up when running MPI with just one MPI process.

**Comment by paulromano** [comment]
Thanks @geogunow. I just put in a fix for this in #867.

**Comment by paulromano** [comment]
Closed with merging of #867.
